ci/unit_tests/intents_csv2json/processExample_test.py

```diff
@@ -0,0 +1,143 @@
+        assert result == expectedResult
+
+    def test_intentWithNestedEntities(self):
+        ''' Test for nested entities'''
+        expectedResult = {
+                "text": "very tall green tree",
+                "mentions": [
+                    {
+                        "location": [
+                            10,
+                            15
+                        ],
+                        "entity": "color"
+                    }
+                ]
+            }
+        result = intents_csv2json.processExample('<plant>very tall <color>green</color> tree</plant>', 'TestIntent', [])
+        assert result == expectedResult
+
+    def test_intentWithMoreNestedEntities(self):
+        ''' Test for nested entities'''
+        expectedResult = {
+                "text": "small brown pine",
+                "mentions": [
+                    {
+                        "location": [
+                            0,
+                            5
+                        ],
+                        "entity": "size"
+                    },
+                    {
+                        "location": [
+                            6,
+                            11
+                        ],
+                        "entity": "color"
+                    }
+                ]
+            }
+        result = intents_csv2json.processExample('<plant><size>small</size> <color>brown</color> pine</plant>', 'TestIntent', [])
+        assert result == expectedResult
+
+    def test_intentWithManyLevelNestedEntities(self):
+        ''' Test for nested entities'''
+        expectedResult = {
+                "text": "hello world, how are you? this is black",
+                "mentions": [
+                    {
+                        "location": [
+                            34,
+                            39
+                        ],
+                        "entity": "color"
+                    }
+                ]
+            }
+        result = intents_csv2json.processExample('<level2>hello <level1>world, how are you? this is <color>black</color></level1></level2>', 'TestIntent', [])
+        assert result == expectedResult
+
+    def test_intentAlreadyExisting(self):
+        ''' Test for nested entities'''
+        expectedResult = None
+        result = intents_csv2json.processExample('<hey>nothing <test>like</test> the above</hey>', 'TestIntent', [{"text": "nothing like the above"}])
+        assert result == expectedResult
+
+    def test_intentAnnontationButEmpty(self):
+        ''' Test for nested entities'''
+        expectedResult = None
+        result = intents_csv2json.processExample('<empty></empty>', 'TestIntent', [])
+        assert result == expectedResult
+
+    def test_intentInvalidAnnotation(self):
+        ''' Test for nested entities'''
+        self.t_exitCode(1, ['#Doing $some <tags> @ which ~ are <not> </in> </not> correct @what?', 'TestIntent', []])
```

Comment:
please try use `t_exitCodeAndLogMessage`

Reply:
all above fixed, thanks for comments